### PR TITLE
Bridged USDC Standard for the OP Stack

### PR DESCRIPTION
**Pull request by krofax**
**Description**

This PR provides a high-level overview of USDC implementations on OP Chains, including the types of USDC representations, deployment options, and considerations for chain operators. 

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

File: pages/builders/chain-operators/features/usdc-implementation-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,245 @@
+---
+title: USDC Implementation Guide
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/usdc-implementation-guide.mdx:2]: Bridged USDC Standard for the OP Stack implementation guide

File: pages/builders/chain-operators/features/usdc-implementation-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+lang: en-US
+description: Learn the two options for implementing Bridged USDC on OP Chain
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/usdc-implementation-guide.mdx:4]: Learn how to deploy the Bridged USDC Standard on your OP Stack chain

File: pages/builders/chain-operators/features/usdc-implementation-guide.mdx [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+---
+
+import { Callout, Steps } from 'nextra/components'
+import Image from 'next/image'
+
+# USDC Implementation Options
+
+OP Chain operators have two main options for implementing Bridged USDC:
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/usdc-implementation-guide.mdx:13]: OP Chain => OP Stack chain operators, there's only 1 option to ship this :D

File: pages/builders/chain-operators/features/usdc-implementation-guide.mdx [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+
+The main differences between the pre-upgrade and post-upgrade stages are:
+
+1.  The USDC representation changes from Bridged USDC to Native USDC.
+2.  The USDC Proxy admin control shifts from the OP Chain operator to Circle.
+3.  The bridging mechanism changes, with the custom bridge being deprecated in favor of Circle's CCTP contracts.
+
+These changes allow for a seamless transition to Native USDC while maintaining the same contract addresses and preserving user balances and integrations.
+
+## Standard Bridged USDC Path
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/usdc-implementation-guide.mdx:78]: Please remove this section

File: pages/builders/chain-operators/features/usdc-implementation-guide.mdx [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+---
+
+import { Callout, Steps } from 'nextra/components'
+import Image from 'next/image'
+
+# USDC Implementation Options
+
+OP Chain operators have two main options for implementing Bridged USDC:
+
+## Bridged-to-Native Upgrade Path
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/usdc-implementation-guide.mdx:15]: Bridged USDC for the OP Stack

Comment (zainbacchus) [reply pages/builders/chain-operators/features/usdc-implementation-guide.mdx:15]: Okay this entire section is wrong. It should only referenced the Bridged USDC Standard for the OP Stack using the directions here: 

https://github.com/defi-wonderland/opUSDC/edit/dev/README.md

It looks like some of this content is from an old document prior to having the above code available for chain operators

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+*   Read the [Bridged USDC Standard](https://www.circle.com/blog/bridged-usdc-standard) from Circle.
+*   For technical details on the deployment and bridging processes, as well as information about important contracts, Read [USDC implementation guide](/builders/chain-operators/features/usdc-implementation-guide).
+
+## FAQs
+
+### What is the StandardBridge?
+
+The StandardBridge enables permissionless transferring of ETH and ERC-20 tokens between L1 and L2.
+
+### Can I deploy a [Bridged USDC Standard by Circle](https://www.circle.com/blog/bridged-usdc-standard) using the StandardBridge?
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:64]: please remove

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+When deciding which USDC implementation to use, consider the following:
+
+*   Do you anticipate needing Native USDC in the future?
+*   Can you allocate resources to implement the Bridged USDC Standard by Circle?
+*   Are you prepared to handle the security responsibilities of the chosen implementation?
+
+## Security Considerations
+
+*   The Bridged USDC Standard for the OP Stack has undergone audits and is recommended for production use.
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:50]: please add legal disclaimer and link to audits

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+*   The Bridged USDC inherits the same security model as the OP Chain rollup.
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:51]: While the Bridged USDC Standard for the OP Stack inherits the same security model as the OP Stack rollup; governance over the upgradable contract would be owned by the chain operator.

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+<Callout type="info">
+    OP chain operators can use Bridged USDC at launch, so they can have it available for their users.
+</Callout>
+
+Chain operators can use the Bridged USDC Standard to:
+
+1.  Deploy Bridged USDC on the OP Stack chain.
+2.  Provide optionality for Circle to seamlessly upgrade Bridged USDC to Native USDC.
+
+## Considerations for Chain Operators
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:40]: Remove

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+1.  **Bridged USDC**: USDC that is bridged from Ethereum (L1) to your OP Chain (L2).
+2.  **Native USDC**: USDC that is issued natively on your OP Chain (L2) by Circle and can be moved between chains using the [CCTP protocol contracts](https://developers.circle.com/stablecoins/docs/cctp-protocol-contract).
+  
+<Callout type="info">
+  *   Bridged USDC: [USDC.e on OP Mainnet](https://basescan.org/token/0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca), USDbC on Base
+  *   Native USDC: [USDC on OP Mainnet](https://basescan.org/token/0x833589fcd6edb6e08f4c7c32d4f71b54bda02913), USDC on Base
+</Callout>
+
+<Callout type="info">
+    OP chain operators can use Bridged USDC at launch, so they can have it available for their users.
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:32]: OP Stack chain operators can use the Bridged USDC Standard at launch, so they can make USDC available for their users.

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+It utilizes the cross-chain messaging of the canonical OP Stack bridge, allowing for easy access to Bridged USDC liquidity across OP Stack chains.
+
+## Types of USDC
+
+There are two main types of USDC representations on OP Chains:
+
+1.  **Bridged USDC**: USDC that is bridged from Ethereum (L1) to your OP Chain (L2).
+2.  **Native USDC**: USDC that is issued natively on your OP Chain (L2) by Circle and can be moved between chains using the [CCTP protocol contracts](https://developers.circle.com/stablecoins/docs/cctp-protocol-contract).
+  
+<Callout type="info">
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:26]: remove

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+## Bridged USDC Standard
+
+The Bridged USDC Standard by Circle provides an efficient and modular solution for expanding the Bridged USDC Standard across the Superchain ecosystem.
+It utilizes the cross-chain messaging of the canonical OP Stack bridge, allowing for easy access to Bridged USDC liquidity across OP Stack chains.
+
+## Types of USDC
+
+There are two main types of USDC representations on OP Chains:
+
+1.  **Bridged USDC**: USDC that is bridged from Ethereum (L1) to your OP Chain (L2).
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:23]: OP Stack chain (L2)

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+It aims to help OP Chain operators understand and implement the most suitable USDC strategy for their needs, with a focus on the Bridged USDC Standard by Circle.
+
+## Bridged USDC Standard
+
+The Bridged USDC Standard by Circle provides an efficient and modular solution for expanding the Bridged USDC Standard across the Superchain ecosystem.
+It utilizes the cross-chain messaging of the canonical OP Stack bridge, allowing for easy access to Bridged USDC liquidity across OP Stack chains.
+
+## Types of USDC
+
+There are two main types of USDC representations on OP Chains:
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:21]: OP Stack chains.

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 14]
```diff
@@ -0,0 +1,76 @@
+---
+title: Bridged USDC Standard on OP Stack
+lang: en-US
+description: This guide explains the different types of USDC representations, their implementations, and considerations for chain operators.
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:4]: This guide explains the different types of USDC representations and how to get the Bridged USDC Standard on your OP Stack chain.

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx (hunk 7 above)
Comment (krofax) [reply pages/builders/chain-operators/features/bridged-usdc-standard.mdx:50]: @zainbacchus To confirm, is this the correct audit link - https://github.com/defi-wonderland/opUSDC/blob/main/audits/spearbit.pdf?

Also what does the legal disclaimer look like, i have no idea what it should be?

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+This explainer provides a high-level overview of USDC implementations on OP Stack, including the types of USDC representations, deployment options, and considerations for chain operators.
+It aims to help OP Stack chains operators understand and implement the most suitable USDC strategy for their needs, with a focus on the Bridged USDC Standard by Circle.
+
+## Bridged USDC Standard
+
+The Bridged USDC Standard by Circle provides an efficient and modular solution for expanding the Bridged USDC Standard across the Superchain ecosystem.
+It utilizes the cross-chain messaging of the canonical OP Stack bridge, allowing for easy access to Bridged USDC liquidity across OP Stack chains.
+
+<Callout type="info">
+  *   Bridged USDC: [USDC.e on OP Mainnet](https://basescan.org/token/0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca), USDbC on Base
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:20]: Please remove this

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+  *   Native USDC: [USDC on OP Mainnet](https://basescan.org/token/0x833589fcd6edb6e08f4c7c32d4f71b54bda02913), USDC on Base
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:21]: Please remove this

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+
+<Callout type="info">
+    OP stack chain operators can deploy Bridged USDC at launch, providing immediate USDC availability for their users. 
+    Importantly, the Bridged USDC Standard allows for a seamless, in-place upgrade to Native USDC if an agreement is later reached between the chain operator and Circle.
+</Callout>
+
+## Security Considerations
+
+*   The Bridged USDC Standard for the OP Stack has undergone [audits](https://github.com/defi-wonderland/opUSDC/blob/main/audits/spearbit.pdf) and is recommended for production use.
+*   The Bridged USDC inherits the same security model as the OP Stack chains rollup.
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:32]: Please remove

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+*   While the Bridged USDC Standard for the OP Stack utilizes the cross domain messenger, governance over the upgradable contract would be owned by the chain operator.
+
+## Next Steps
+
+*   Read the [Bridged USDC Standard](https://www.circle.com/blog/bridged-usdc-standard) from Circle.
+*   Read the [Bridged USDC Standard Factory Disclaimer](https://github.com/defi-wonderland/opUSDC?tab=readme-ov-file#bridged-usdc-standard-factory-disclaimer).
```
Comment (zainbacchus) [inline pages/builders/chain-operators/features/bridged-usdc-standard.mdx:38]: Disclaimer needs to be on this page specifically.

File: pages/builders/chain-operators/features/bridged-usdc-standard.mdx (hunk 17 above)
Comment (krofax) [reply pages/builders/chain-operators/features/bridged-usdc-standard.mdx]: @zainbacchus Should i remove all the points in here, or just "The Bridged USDC inherits the same security model as the OP Stack chains rollup."?

**Review by zainbacchus — APPROVED** [review]
lgtm

**Pull request merged**